lib/v4/migration-helpers/update-application-folder-structure.js

```diff
@@ -0,0 +1,23 @@
+const { resolve, join, basename } = require('path');
+const fs = require('fs-extra');
+const { logger } = require('../../global/utils');
+
+const moveDir = async (appPath, folder, srcDir) => {
+  try {
+    await fs.copy(join(appPath, folder), srcDir);
+  } catch (error) {
+    logger.info(`${basename(appPath)}/${folder} not found, skipping...`);
+  }
+};
```

Comment:
Just one last thing if you don't mind. I think there is an issue here because it should copy to the folder, or in the case of admin it should copy to `admin/extenstions`.

```suggestion
const moveDir = async (appPath, folder, srcDir) => {
  const destination =
    folder === 'admin' ? `${join(srcDir, folder)}/extenstions` : join(srcDir, folder);
		
  try {
    await fs.copy(join(appPath, folder), destination);
  } catch (error) {
    logger.info(`${basename(appPath)}/${folder} not found, skipping...`);
  }
};
```